### DISCUSSION TITLE
AppImage for Ubuntu 18.04

### DISCUSSION THREAD

**Discussion by juanluisbaptiste** [bgallois/OpenJournal#8] [General]
Hi!

Have you thought on releasing an AppImage file for Ubuntu that supports 18.04 version? when I run the latest version on that Ubuntu I get this error:

`./OpenJournal-x86_64.AppImage: /lib/x86_64-linux-gnu/libm.so.6: version GLIBC_2.29 not found (required by /tmp/.mount_OpenJoBvkPLz/lib/libqalculate.so.20)
`

I'm stuck at work with that version with my work laptop and I cannot update it :(

**Comment by bgallois** [comment]
Hi,
As GitHub Actions doesn't provide Ubuntu 18.04 anymore, I'm not releasing the AppImage for Ubuntu<20.04.
That being said, OpenJournal does not include many dependencies and for Ubuntu
```
sudo apt-get install qtbase5-dev qtwebengine5-dev libqalculate-dev
git clone https://github.com/bgallois/OpenJournal.git
cd OpenJournal
qmake OpenJournal.pro
make
sudo make install
```
and OpenJournal should be integrated fully in your system.

**Comment by juanluisbaptiste** [comment]
Hi @bgallois , excellent, I will try that, thank you.
